examples/syn-flood/main.rs

```diff
@@ -45,16 +47,23 @@ fn install(qs: HashMap<String, PortQueue>) -> impl Pipeline {
     .send(qs["eth1"].clone())
 }
 
+fn print_stats() {
+    let mut observer = YamlBuilder::new().build();
+    metrics::global().controller().observe(&mut observer);
+    println!("{}", observer.drain());
+}
+
 fn main() -> Result<()> {
     let subscriber = fmt::Subscriber::builder()
-        .with_max_level(Level::DEBUG)
+        .with_max_level(Level::INFO)
         .finish();
     tracing::subscriber::set_global_default(subscriber)?;
 
     let config = load_config()?;
     debug!(?config);
 
     Runtime::build(config)?
-        .add_periodic_pipeline_to_core(1, install, Duration::from_millis(100))?
+        .add_periodic_pipeline_to_core(1, install, Duration::from_millis(50))?
```

Comment:
any reasoning for this switch other than just general too long duration?

Reply:
Want to see packets being dropped so we get some more stats.

Reply:
and confirmed.